life/modernized/11-chapter-11.adoc

```diff
@@ -264,7 +277,8 @@ has many ways of transforming himself--even under the most upright intentions,
 and in the truest sincerity of desire,
 to be dedicated to the service of Him whom to know (from the deceit) is life eternal.`'
 
-Therefore, methinks,
+Therefore, methinks, 
+//Do you want this changed to "I think"....I just happened to see it.
```

Comment:
uhh... let's leave it for now, i made a note to discuss it with jason, and i'll change it if we decide.  good question!